src/xdocs/config_misc.xml

```diff
@@ -69,6 +88,23 @@
   &lt;property name=&quot;javaStyle&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example:
+        </p>
+        <source>
+public class MyClass {
+  int[] nums; // violation
+  String strings[]; // OK
+
+  char[] toCharArray() { // OK (method return types not checked for C style)
```

Comment:
this is not consistent behavior.

if `int[] nums; // violation` then `char[] toCharArray()` should be violation.

Reply:
moved to https://github.com/checkstyle/checkstyle/pull/6428#discussion_r257523668